LICENSE.md

```diff
@@ -1,6 +1,7 @@
 The MIT License (MIT)
 
 Copyright (c) Taylor Otwell
+Copyright (c) Jonas Staudenmeir
```

Comment:
I don't think it's common to add to this file on contributions to open source projects. Would we add everyone that has ever contributed to Dusk?

Reply:
We don't have to change the license file, a copyright notice in `ChromeDriverCommand.php` would be sufficient.

Reply:
@staudenmeir I've pushed a commit for this, does this look good to you? https://github.com/laravel/dusk/commit/7e5414b488bbf1e59bbfa77bd0a284cc228678f8

Reply:
Please also include the package URL.

Reply:
Done!

Reply:
Thanks!